README.md

```diff
@@ -81,6 +81,15 @@ Trace trace = tracer.get("X-Trace-ID"); // this is a live-view that can be a sha
 entity.setLastModifiedBy(trace.getValue());
 ```
 
+By default there can only be one trace active at a time. Sometimes it can be useful to *stack* traces:
+
+```java
+Tracer tracer = Tracer.builder()
+    .stacked(true)
```

Comment:
Lombok doesn't support it and I don't want to implement the whole Builder myself.